### DISCUSSION TITLE
export cookie

### DISCUSSION THREAD

**Discussion by mh292929** [xtekky/gpt4free#2317] [Q&A]
hello Can you explain how to get cookie output from Bing? bing.com/chat is no longer available

**Comment by Lamparter** [comment]
Hi @MH140000, could you please clarify what you mean? If you want to view the cookies of a website, you can do so via the Inspect tool, (<kbd>F12</kbd>) or by pressing the lock button in the address bar and pressing on **View cookies**.

###### If you found this useful, feel free to mark it as an answer!

**Comment by mh292929** [comment]
I need Gemini and bing cookie for g4f

**Reply by Lamparter** [reply]
Sorry? What do you mean?

**Comment by mh292929** [comment]
How to export _u cookie for this project?

**Reply by Lamparter** [reply]
@MH140000 if you had read what I said, you would know that you can find the cookies of a website by: 
> If you want to view the cookies of a website, you can do so via the Inspect tool, (<kbd>F12</kbd>) or by pressing the lock button in the address bar and pressing on **View cookies**.
